Changes

```diff
@@ -39,6 +39,13 @@ Working version
   (Enguerrand Decorne, KC Sivaramakrishnan and Tom Kelly,
   review by Anil Madhavapeddy and Gabriel Scherer)
 
+- #11091: Allocate sigaltstat via mmap.
```

Comment:
Could you mention that this is an openBSD fix?
I would also suggest to move the change entry to the bug fix section.